canonical-websites/www.canonical.com
# Convert /partners to Vanilla v1 styles

## Done

Upgrade /partners to v1

## QA

- Pull code
- Check markup at 
`templates/partners/index.html`
`templates/partners/contact-us.html`
`templates/partners/thank-you.html`
- Run `./run serve --watch`
- Check page locally at 
http://0.0.0.0:8002/partners
http://0.0.0.0:8002/contact-us
http://0.0.0.0:8002/thank-you

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/31

## Demo

http://www.canonical.com-homepage-polish.demo.haus/partners
